docs/funnel-stages.md

```diff
@@ -0,0 +1,144 @@
+# Open Source Funnel Stages<a id="funnel-stages"></a>
+
+Scarf assigns a point value, or score, to companies based on event activity, such as documentation views or artifact downloads. This score considers the recency, frequency of event activity, and Event Importance, combining the benefits of traditional lead scoring and intent data. That point value is used to determine the **Funnel Stage**, which helps to clearly identify a company’s readiness for various types of marketing or sales engagement. 
```

Comment:
We want to be a little careful here because there are two different notions of "points" here. There are points that we assign to actions that get leads to their next funnel stage, but there is separately points that go into "Scarf Score" which represents our notion of their actionability. Further down the funnel doesn't necessarily mean more actionable, as we might disqualify companies that don't fit the customer's target profile, etc.

Reply:
What do you think about this: 

Scarf tracks event activity, such as documentation views or artifact downloads, and considers the recency, frequency of activity, and Event Importance, combining the benefits of traditional lead scoring and intent data and formulating a Funnel Stage.

Reply:
Maybe something more like:

"Scarf analyzes how companies interact with your open source project to infer their progress towards adopting it. Funnel Stages represent the portion of the user journey that best describes any given company or lead, from the moment they learn of your project, to when they deploy it to production, and beyond."

You could mention custom intent signals here but it actually feels like more of a detail than a top-line value proposition.